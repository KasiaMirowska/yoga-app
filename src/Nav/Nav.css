nav {
    padding: 10px;
    background-color: rgb(221, 243, 233)  ;
    /* background-color: rgb(201, 220, 231); */
}