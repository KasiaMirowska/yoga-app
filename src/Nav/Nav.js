import React from 'react';
import { Link } from 'react-router-dom';
import Login from '../Login/LoginPage';
import Register from '../Register/Register';


export default function Nav() {
    return (
        <div>
            <Link to={'/login'} className='link' >
             <Login />
           </Link>
           <Link to={'/register'} className='register'>
             <Register />
           </Link>
        </div>
    )
}