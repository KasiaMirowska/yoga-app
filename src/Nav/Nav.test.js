import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import Nav from './Nav';
import {BrowserRouter as BR} from 'react-router-dom';

describe('Nav component', () => {
    it('renders without crashing', () => {
         const div = document.createElement('div');
         ReactDom.render(<BR><Nav /></BR>, div);
         ReactDom.unmountComponentAtNode(div);
    });
    it('renders UI as expected', () => {
        const nav = renderer.create(<BR><Nav /></BR>)
        expect(nav.toJSON()).toMatchSnapshot();
    })
})