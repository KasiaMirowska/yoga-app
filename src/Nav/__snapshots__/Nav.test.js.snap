// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders UI as expected 1`] = `
<div
  className="nav"
>
  <a
    className="link"
    href="/login"
    onClick={[Function]}
  >
    Login
  </a>
  <a
    className="register"
    href="/register"
    onClick={[Function]}
  >
    Register
  </a>
</div>
`;
